content/realtime/authentication.textile

```diff
@@ -77,42 +77,110 @@ Here is a straightforward example of using Basic Authentication to connect to Ab
+Tokens are authentication credentials that are short-lived, and therefore they may more readily be distributed to clients where there is a risk of compromise. Tokens may also be issued with a particular scope - such as a limited set of "access rights or capabilities":/core-features/authentication#capabilities-explained or being limited to use by a specific "<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span> identity":#identified-clients - and therefore token-based authentication provides the flexibility to implement access and identity control policies in the application. See the "Token Authentication documentation":/core-features/authentication#token-authentication for more details. To initialize the "Realtime library":/rest/usage#constructor to a previously obtained "@Ably Token@":#token-details or "@Ably JWT@":/core-features/authentication#ably-jwt, set the <span lang="ruby">@:token@ or @:token_details@</span><span lang="python">@token@ or @token_details@</span><span lang="csharp">@Token@ or @TokenDetails@</span><span lang="default">@token@ or @tokenDetails@</span> attribute of "@ClientOptions@":/rest/usage#client-options to the token.
 
-Below is a rather contrived yet straightforward example that instances a Realtime library using "Token Authentication":/core-features/authentication#token-authentication with a means to reissue tokens as required. Typically, in a browser environment, the <span lang="default">"@authUrl@":#auth-options</span><span lang="csharp">"@AuthUrl@":#auth-options</span> provided would be a relative URL to a local endpoint that is used to issue tokens to trusted clients. Client requests can, for example, be trusted based on session cookies. For non-browser clients, an authentication callback is preferred thus relying on your application to communicate securely with your own servers to obtain a token.
+h3(#auth-url). Token refresh using @authUrl@
 
-```[jsall](code-editor:realtime/auth-token-callback)
-  var realtime = new Ably.Realtime({ authUrl: 'http://my.website/auth' });
+Below is an example that instances a Realtime library using "Token Authentication":/core-features/authentication#token-authentication with a means to reissue tokens as required. 
+
+Typically, in a browser environment, the <span lang="default">"@authUrl@":#auth-options</span><span lang="csharp">"@AuthUrl@":#auth-options</span> provided would be a relative URL to a local endpoint that is used to issue tokens to trusted clients. Client requests can, for example, be trusted based on session cookies.
```

Comment:
I think "local endpoint" here is incorrect (local to what?), I think it should be "endpoint under the same origin"

Reply:
Reworded to be correct, "endpoint under the same origin" is what I actually meant!